src/Components/AddToListDropMenu.js

```diff
@@ -154,7 +154,6 @@ export default function AddToListDropMenu({ anime, variant, selected }) {
           onClick={(e) => {
             handleToggle();
             e.preventDefault();
-            e.stopPropagation();
```

Comment:
Nice fix, and so simple!  I did think it was a little weird to be able to get a bunch of those boxes open at the same time